prowler/lib/outputs/jira/jira.py

```diff
@@ -98,6 +100,10 @@ class Jira:
     _client_id: str = None
     _client_secret: str = None
     _access_token: str = None
+    _client_mail: str = None
+    _api_token: str = None
```

Comment:
For me `user_mail`/`mail` and `token` are more appropriate here.

Reply:
I changed it from `client_mail` -> `user_mail` but for the token, I like `api_token` since we are already using token var in other parts of the code and `api_token` sounds more distinctive for me

Reply:
Thanks for the clarification!